jellyfin_apiclient_python/api.py

```diff
@@ -474,6 +581,26 @@ def send_request(self, url, path, method="get", timeout=None, headers=None, data
 
         return request_method(url, **request_settings)
 
+    # TODO: Quick connect is not responding to the requests.
```

Comment:
Again, any idea why this may be an issue?

Reply:
I'm sorry the Jellyfin is like a blackbox to me and I only make some calls to see if the server can return me some information. The quick connect is not working at all. By the way, my initial thought is to use an authorization token to create a session rather than use username, password or PIN to log in. If you know it, please send me a link to the API call. Thanks!